package.json

```diff
@@ -42,6 +42,7 @@
     "react-router-dom": "5.3.0",
     "react-scripts": "4.0.3",
     "react-share": "^4.4.0",
+    "react-toastify": "^8.2.0",
```

Comment:
We have a toast component inside react-bootstrap, will change it over to that now so we don't rely on an extra dependency